### PR TITLE
Minor changes to user guides

### PR DESCRIPTION
**Pull request by CloneWith**
- Add common whistle signal patterns
- The camera supports pinching to zoom in / out
- Add warnings to Flashlight tool

### CODE REVIEW

File: guides/en-US/guide_tool_flashlight.txt
Comment (kylecorry31) [inline guides/en-US/guide_tool_flashlight.txt:1]: Can you change this to: Please note that the strobe feature contains flashing lights, which may be dangerous for some users.



I can't find anything about this causing damage to a camera - if that's true, can you let me know where you found that information?

Comment (CloneWith) [reply guides/en-US/guide_tool_flashlight.txt:1]: I had an old device's flashlight damaged after keeping it on for a few hours 😂
nevertheless the hardware has been improved a lot so far, and this happened nearly about 10 years ago, so I think it rarely happens now.

Comment (kylecorry31) [reply guides/en-US/guide_tool_flashlight.txt:1]: 😆 that's a long time to keep it on, I think nowadays devices are more likely to overheat and shutdown than have damage. 

But I guess to be on the safe side, maybe this messaging would work instead: "Please note that the strobe feature contains flashing lights, which may be dangerous for some users. Prolonged use of the strobe may cause excess battery drain or damage to your camera's flash."

**Comment by kylecorry31** [comment]
Looks good, thanks!

**Pull request merged**